config/initializers/erdbeere_connection.rb

```diff
@@ -0,0 +1,5 @@
+ERDBEERE_API_URL = ENV.fetch("ERDBEERE_API")
+
+ERDBEERE_CONNECTION = Faraday.new(url: ERDBEERE_API_URL) do |conn|
+  conn.adapter(Faraday.default_adapter)
+end
```

Comment:
In production, how does MaMpf communicate with the Erdbeere server? Is Erdbeere just available on localhost or also available to the word-wide web. In the latter case, how does MaMpf authenticate at Erdbeere? I'm wondering in particular since Erdbeere itself has a frontend which allows to log in.

Reply:
This has been discussed in Basecamp.